game/regex.hh

```diff
@@ -0,0 +1,16 @@
+
+#pragma once
+
+// Note: One could expect to have boost regex removed, but for some obscure
+// reason, std::regex code crash on Fedora 30 (see bug #476). Thus, until FC is
+// fixed or deprecated, may be eeded to keep this for a little while
+
+#ifdef USE_BOOST_REGEX
+#include <boost/regex.hpp>
+using regex = boost::regex;
+namespace regex_constants = boost::regex_constants;
+#else
+#include <regex>
+using regex = std::regex;
+namespace regex_constants = std::regex_constants;
+#endif
```

Comment:
Shouldn't this import `regex_replace` too?

Reply:
yes and regex_search as well... let me figure out how this can compile...

Reply:
well actually this is fine, thanks to the argument-dependent name lookup. Here should be ok, as there is no abiguity between boost and std (when using boost regex, all is boost::regex, when using std all is std::regex)

Reply:
Seems this is 'resolved'? since tronic thumbed-up the response?